strongbox-resources/strongbox-storage-resources/strongbox-storage-api-resources/src/main/resources/etc/conf/strongbox.xml

```diff
@@ -13,6 +13,18 @@
 
     <storages>
 
+        <storage id="storage-npm-common">
```

Comment:
Would you mind renaming the storage to `storage-common-npm`? (All the other storages are named similarly).

Reply:
ok